repo/packages/N/nexus/0/config.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Package should install with defaults. Please make sure it does even if you have some required fields. Add defaults there.